eos/form-factors/parametric-bsz2015.hh

```diff
@@ -195,15 +257,9 @@ namespace eos
             // use equation of motion to remove f_0(0) as a free parameter
             std::array<UsedParameter, 2> _a_fz;
 
-            const double _mB, _mB2, _mP, _mP2;
-            const double _tau_p, _tau_0;
-            const double _z_0;
-
-            static double _calc_tau_0(const double & m_B, const double & m_P);
-
-            complex<double> _calc_z(const complex<double> & s) const;
+            const BSZ2015FormFactorTraits<Process_, PToP> _traits;
 
-            double _calc_z(const double & s) const;
+            const double _mB, _mB2, _mP, _mP2;
```

Comment:
Also here, why are these not covered by the traits?

Reply:
Again there are just here for convenience, but they are constructed from the traits. Let me know if you agree or if you prefer removing them entirely